src/components/sections/LatticeView.tsx

```diff
@@ -88,52 +87,26 @@ const LatticeView = ({ compilationResult }: LatticeViewProps): JSX.Element => {
                 />
             </Flex>
 
-            {showCompilationText ? (
+            {showCompilationText && (
                 <Flex gap={10} justifyContent={"center"} pt={0} pb={4} flexWrap={"wrap"}>
-                    <Box
-                        className="box-hover"
-                        textAlign="center"
-                        borderWidth="4px"
-                        borderRadius="xl"
-                        boxShadow={"xl"}
```

Comment:
ChakraUI doesn't seem to mind if the class attributes are wrapped in curly braces or just quotes. I'm not sure if our linters will be able to make it consistent, but I think if we can, we should remove the curly braces where possible to be consistent

Reply:
This doesn't apply to this PR, but just in general

Reply:
I trust in the linters for these things